File: RJ-101/sp-1/Evaluation-1/Eve-1/Employee.js
class Employee{
    constructor(){

    }

    Employee(json){
        this.currentPage=json.currentPage;
        this.data=json.data;
    }

    get currentPage(){
        return this.#currentPage;
    }

    get currentPage(){
        return this.#data;
    }

   
}

File: RJ-101/sp-rep-3/Junit testing/Junit_testing/src/main/java/com/push/models/Employee.java
package com.push.models;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Entity
@AllArgsConstructor
@NoArgsConstructor
@Data
@ToString
public class Employee {
	
	@Id
	@GeneratedValue(strategy = GenerationType.AUTO)
	private Integer empId;
	private String name;
	private String department;

}
